Userland/Applications/FontEditor/MainWidget.cpp

```diff
@@ -113,8 +114,11 @@ ErrorOr<void> MainWidget::create_actions()
         Optional<DeprecatedString> open_path = GUI::FilePicker::get_open_filepath(window(), {}, "/res/fonts/"sv);
         if (!open_path.has_value())
             return;
-        if (auto result = open_file(open_path.value()); result.is_error())
+        auto result = open_file(open_path.value());
+        if (result.is_error())
             show_error(result.release_error(), "Opening"sv, LexicalPath { open_path.value() }.basename());
+        else
+            GUI::Application::the()->set_most_recently_open_file(MUST(String::from_deprecated_string(open_path.value())));
```

Comment:
`set_most_recently_open_file()` can be consolidated into `open_file()` after initialization.
Other apps set a recent item after saving as too.